avalon/maya/pipeline.py

```diff
@@ -522,6 +522,9 @@ def remove(container):
     fname = cmds.referenceQuery(reference_node, filename=True)
     cmds.file(fname, removeReference=True)
 
+    if cmds.objExists(node):
```

Comment:
Referenced Alembic animation, tried `cmds.detele(node)`.
Result when removing through Manager:

```python
# Traceback (most recent call last):
#   File "C:\Users\User\Documents\development\avalon\core\avalon\tools\cbsceneinventory\app.py", line 96, in <lambda>
#     lambda: self.show_remove_warning_dialog(items))
#   File "C:\Users\User\Documents\development\avalon\core\avalon\tools\cbsceneinventory\app.py", line 250, in show_remove_warning_dialog
#     host.remove(item)
#   File "C:\Users\User\Documents\development\avalon\core\avalon\maya\pipeline.py", line 526, in remove
#     
# ValueError: No object matches name: shot1_01__test_001__CON
```
Existence of node cannot be guaranteed.